azkaban-common/src/main/java/azkaban/project/NodeBeanLoader.java

```diff
@@ -107,6 +107,13 @@ private void validateSchedule(final FlowTriggerBean flowTriggerBean) {
                 .isValidExpression(scheduleMap.get(FlowTriggerProps.SCHEDULE_VALUE)),
             "flow trigger schedule value must be a valid cron expression");
 
+    final String cronExpression = scheduleMap.get(FlowTriggerProps.SCHEDULE_VALUE).trim();
+    final String[] cronParts = cronExpression.split("\\s+");
+
+    Preconditions
```

Comment:
Do you have any sort of integration test suite? How would it affect the evaluation time length after this change?

Reply:
the unit tests covered the illegal case. Time length is trivial since it's just regular expression matching cost for a string.